Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.py

```diff
@@ -1341,6 +1345,97 @@ def get_policy_details_request(policy_id):  # pragma: no cover
     return res
 
 
+def create_instaquery():
+    query_args = demisto.args()
+    name = query_args['name']
+    description = query_args['description']
+    artifact = query_args['artifact']
+    match_value_type = query_args['match_value_type']
+    match_values = query_args['match_values'].split(",")
+    match_type = query_args['match_type']
```

Comment:
Please use `query_args.get('arg_name')` (where 'arg_name' is the relevant argument name) and not `query_args['arg_name']`.

Reply:
Yes, updated to use .get()